.github/scripts/conventional-commit-pr-title-reminder/utils.js

```diff
@@ -4,15 +4,15 @@ exports.updateComment = updateComment;
 async function updateComment({ github, context }) {
   const githubUsername = context.payload.sender.login;
   const comment = `${CONVENTIONAL_COMMIT_REMINDER_COMMENT_IDENTIFIER}
-@${githubUsername} You have modified embedding SDK code. Please make sure the PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) style.
+@${githubUsername} You have modified embedding SDK package code. Please make sure the PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) style.
 Here are all the supported types that will be shown in the changelog:
 - \`feat\`
 - \`fix\`
 - \`perf\`
 
 These types \`docs\`, \`style\`, \`refactor\`, \`test\`, \`build\`, and \`ci\` are also encouraged for non-changelog related tasks.
 
-Please also make sure to include \`sdk\` scope, otherwise, the changelog will not include this task.
+Please also make sure to include \`sdk-package\` scope, otherwise, the changelog will not include this task.
```

Comment:
Can we briefly tell people what is a "sdk-package" here? Unless you know about the hosted bundles project it's not obvious that there are two sides of the SDK.

Reply:
ok, let's keep it as-is